### PR TITLE
Update QEC Containers and PublicCloud configs

### PR DESCRIPTION
**Pull request by pdostal**
Let's get the QEC Containers and Public Cloud configs up-to-date.

Have a good Monday!

### CODE REVIEW

File: _review/qec-publiccloud.toml [hunk 1]
```diff
@@ -47,4 +49,62 @@ Params = { flavor = "GCE-Updates", build = "%yesterday%-1" }
...
+[[Groups]]
+Name = "Server-DVD-Updates"
+Params = { flavor ="Server-DVD-Updates", groupid = "427", build = "" }
+MaxLifetime = 86400
+
+# Incidents
 
+[[Groups]]
+Name = "Azure-BYOS-Incidents"
+Params = { flavor = "Azure-BYOS-Incidents", build = "%yesterday%-1" }
```
Comment (asmorodskyi) [inline _review/qec-publiccloud.toml]:
```suggestion
Params = { flavor = "Azure-BYOS-Incidents", build = "" }
```

incidents not applying to this pattern

**Review by asmorodskyi — APPROVED** [review]
LGTM ( after fixing build pattern for Incidents )

File: _review/qec-publiccloud.toml (hunk 1 above)
Comment (pdostal) [reply _review/qec-publiccloud.toml]: Fixed. Thank you Anton

File: _review/qec-publiccloud.toml [hunk 2]
```diff
@@ -47,4 +49,62 @@ Params = { flavor = "GCE-Updates", build = "%yesterday%-1" }
 [[Groups]]
 Name = "GCE-BYOS-Updates"
 Params = { flavor = "GCE-BYOS-Updates", build = "%yesterday%-1" }
+[[Groups]]
+Name = "GCE-CHOST-BYOS-Updates"
+Params = { flavor = "GCE-CHOST-BYOS-Updates", build = "%yesterday%-1" }
+
+[[Groups]]
+Name = "Server-DVD-Updates"
```
Comment (grisu48) [inline _review/qec-publiccloud.toml]: Server-DVD-Updates? Does that even make sense for Publiccloud?

Comment (grisu48) [reply _review/qec-publiccloud.toml]: Let's rename the name to something more descriptive:

```suggestion
Name = "Publiccloud CLI tools"
```

File: _review/qec-publiccloud.toml [hunk 3]
```diff
@@ -47,4 +49,62 @@ Params = { flavor = "GCE-Updates", build = "%yesterday%-1" }
...
+Params = { flavor ="Server-DVD-Updates", groupid = "427", build = "" }
+MaxLifetime = 86400
+
+# Incidents
 
+[[Groups]]
+Name = "Azure-BYOS-Incidents"
+Params = { flavor = "Azure-BYOS-Incidents", build = "" }
```
Comment (grisu48) [inline _review/qec-publiccloud.toml:67]: Please add `MaxLifetime = 86400` to all incidents otherwise very old builds are displayed as well.

**Review by grisu48 — CHANGES_REQUESTED** [review]
Please add `MaxLifetime` to incidents, otherwise too old test runs spam the view.

File: _review/qec-publiccloud.toml (hunk 2 above)
Comment (pdostal) [reply _review/qec-publiccloud.toml]: May be 😉

Comment (pdostal) [reply _review/qec-publiccloud.toml]: Done.

File: _review/qec-publiccloud.toml (hunk 3 above)
Comment (pdostal) [reply _review/qec-publiccloud.toml:67]: Done.

**Review by grisu48 — APPROVED** [review]
LGTM, thanks for the update!

**Pull request merged**